unified-codes/libs/ui/src/lib/components/entity/organisms/EntityBrowser.stories.tsx

```diff
@@ -1,32 +1,340 @@
+      columns={columns}
+      onSort={onSort}
+      orderDesc={orderDesc}
+      orderBy={orderBy}
+    />
+  ), [classes, columns, onSort, orderDesc, orderBy]);
+
+  const entityTableRows = React.useMemo(() => (
+    <React.Fragment>
+      {
+        data.map((entity: IEntity, index: number) => (
+          <EntityTableRow
+            classes={{
+              root: index % 2 ? classes?.table?.rowPrimary : classes?.table?.rowSecondary,
+              cell: classes?.table?.cell
+            }}
+            columns={columns}
+            entity={entity}
+            key={entity.code}
+          ></EntityTableRow>
+        ))
+      }
+    </React.Fragment>
+  ), [classes, columns, data]);
+    
+  const entityBrowserTable = React.useMemo(() => (
+    <EntityTable 
+      classes={classes?.table}
+      header={entityBrowserTableHeader}
+      rows={entityTableRows}
+    />
+  ), [classes, entityBrowserTableHeader, entityTableRows]);
+
+  const entityBrowserTablePagination = React.useMemo(() => (
+    <TablePagination
+      classes={{ root: classes?.tablePagination?.root }}
+      rowsPerPageOptions={rowsPerPageOptions}
+      component="div"
+      count={count}
+      rowsPerPage={rowsPerPage}
+      page={page}
+      onChangePage={onChangePage}
+      onChangeRowsPerPage={onChangeRowsPerPage}
+    />
+  ), [classes, rowsPerPageOptions, count, rowsPerPage, page, onChangePage, onChangeRowsPerPage]);
+
+  // Search bar.
+
+  const { 
+    input,
+    label,
+    setInput,
+  } = useSearchBarState();
+
+  const onChange = React.useCallback((input: string) => setInput(input), [setInput]);
+  const onSearch = React.useCallback((term: string) => setFilterBy(term), [setFilterBy]);
+  const onClear = React.useCallback(() => { setInput(''); setFilterBy(''); }, [setInput, setFilterBy]);
+
+  const entityBrowserSearchBar = React.useMemo(() => (
+    <SearchBar 
+      classes={classes.searchBar}
+      input={input}
+      label={label} 
+      onChange={onChange}
+      onClear={onClear}
+      onSearch={onSearch}
+    />
+  ), [classes, input, label, onChange, onClear, onSearch]);
+
+  // Type filter.
+
+  const { types } = useTypeFilterState();
+
+  const entityBrowserTypeFilter = React.useMemo(() => (
+    <EntityTypeFilter 
+      classes={{ 
+        root: classes?.typeFilter?.root, 
+        toggleButtonActive: classes?.typeFilter?.toggleButtonActive,
+        toggleButtonInactive: classes?.typeFilter?.toggleButtonInactive,
+        toggleButtonGroup: classes?.typeFilter?.toggleButtonGroup,
+      }}
+      types={types} 
+    />
+  ), [classes, types]);
+
+  // Browser.
+
+  return (
+    <EntityBrowser 
+      classes={{ 
+        root: classes?.browser?.root,
+        tableContainer: classes?.browser?.tableContainer, 
+        typeFilterContainer: classes?.browser?.typeFilterContainer,
+        searchBarContainer: classes?.browser?.searchBarContainer,
+        tablePaginationContainer: classes?.browser?.tablePaginationContainer  
```

Comment:
Thinking it might be more atomic to refactor pagination as part of the table component. Since table and pagination share state, this could help avoid splitting shared state and have the added bonus of making `EntityBrowser` only need component props for `table`, `toggleBar`, `searchBar`.

OR, if we want to be super cool, we could add an entity molecule, e.g. replace `EntityTypeFilter` with something called `EntityTableFilter` (but better name that you guys can think up) which wraps `ToggleBar` and `SearchBar`. Too far?